packages/dcos-launch/extra/launch/azure.py

```diff
@@ -61,11 +61,15 @@ def test(self, info, test_cmd):
         launch.util.check_testable(info)
         details = self.describe(info)
         test_host = details['master_fqdn']
+        master_list = [m['private_ip'] for m in details['masters']]
```

Comment:
This bit of code would be a little clearer if this was called `master_private_ips`. The `master_list=master_list` below doesn't make it obvious that we're passing private IPs.